### PR TITLE
Banner manager

### PR DESCRIPTION
**Pull request by torvista**
Minor code tweaks, no functionality changes yet.
To squash after being reviewed.

### CODE REVIEW

File: admin/banner_manager.php [hunk 1]
```diff
@@ -142,7 +142,7 @@
           'status' => $status,
           'banners_open_new_windows' => $banners_open_new_windows,
           'banners_on_ssl' => $banners_on_ssl,
-          'banners_sort_order' => (int)$banners_sort_order,
```
Comment (scottcwilson) [inline admin/banner_manager.php:145]: I don't think you should remove this.

File: admin/banner_manager.php [hunk 2]
```diff
@@ -374,7 +374,7 @@ function popupImageWindow(url) {
             <?php echo zen_draw_label(TEXT_BANNERS_GROUP, 'banners_group', 'class="col-sm-3 control-label"'); ?>
             <div class="col-sm-9 col-md-6">
               <?php echo zen_draw_pull_down_menu('banners_group', $groups_array, $bInfo->banners_group, 'class="form-control" id="banners_group"'); ?><br>
-              <p><?php echo TEXT_BANNERS_NEW_GROUP; ?></p><?php echo zen_draw_input_field('new_banners_group', '', 'class="form-control" id="new_banners_group"', ((count($groups_array) > 0) ? false : true)); ?>
```
Comment (scottcwilson) [inline admin/banner_manager.php:377]: Can the last parameter just be 
```
(count($groups_array) == 0)
```

**Review by scottcwilson — CHANGES_REQUESTED** [review]

File: admin/banner_manager.php (hunk 1 above)
Comment (torvista) [reply admin/banner_manager.php:145]: Even though it's already an integer from line 107?
`$banners_sort_order = (int)$_POST['banners_sort_order'];`

File: admin/banner_manager.php (hunk 2 above)
Comment (torvista) [reply admin/banner_manager.php:377]: Yes. Much better.

**Comment by torvista** [comment]
squashed

**Pull request merged**